recipes/librealsense/all/patches/patch-json-include.patch

```diff
@@ -0,0 +1,11 @@
+--- src/l500/l500-serializable.cpp
++++ src/l500/l500-serializable.cpp
+@@ -2,7 +2,7 @@
+ //// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+ 
+ #include "l500-serializable.h"
+-#include "../../../third-party/json.hpp"
++#include "../../third-party/json.hpp"
+ 
```

Comment:
I have 2 questions about this:
- is this a bug in their code as the relative path is wrong
- what 3rd party json lib is used? Does conan have it already?

Reply:
I think it's the excessive use of `PROJECT_SOURCE_DIR` +> to realative path + the CCI cmake wrapper https://github.com/IntelRealSense/librealsense/search?q=PROJECT_SOURCE_DIR

Yes but do done have anything old
https://github.com/IntelRealSense/librealsense/blob/34fc284d537a4b873a37b737a61f1f1da92dbb60/third-party/json.hpp#L4

Reply:
We have 3.4-3.9. I guess that's too recent and breaking the api?

Reply:
yep!

There are a lot of 3rd party that are embedded, I am not sure it thats a blocker 🚫